includes/class-wc-stripe-webhook-handler.php

```diff
@@ -344,12 +344,17 @@ public function process_webhook_dispute( $notification ) {
 
 		$this->set_stripe_order_status_before_hold( $order, $order->get_status() );
 
-		$message = sprintf(
-		/* translators: 1) HTML anchor open tag 2) HTML anchor closing tag */
-			__( 'A dispute was created for this order. Response is needed. Please go to your %1$sStripe Dashboard%2$s to review this dispute.', 'woocommerce-gateway-stripe' ),
-			'<a href="' . esc_url( $this->get_transaction_url( $order ) ) . '" title="Stripe Dashboard" target="_blank">',
-			'</a>'
-		);
+		$needs_response = in_array( $notification->data->object->status, [ 'needs_response', 'warning_needs_response' ], true );
+		if ( $needs_response ) {
+			$message = sprintf(
+			/* translators: 1) HTML anchor open tag 2) HTML anchor closing tag */
+				__( 'A dispute was created for this order. Response is needed. Please go to your %1$sStripe Dashboard%2$s to review this dispute.', 'woocommerce-gateway-stripe' ),
+				'<a href="' . esc_url( $this->get_transaction_url( $order ) ) . '" title="Stripe Dashboard" target="_blank">',
+				'</a>'
+			);
+		} else {
+			$message = __( 'A dispute was created for this order. It was closed as lost or accepted.', 'woocommerce-gateway-stripe' );
```

Comment:
```suggestion
			$message = __( 'A dispute was created for this order.', 'woocommerce-gateway-stripe' );
```

When the dispute does not need any action/response, it could be lost or won and it could also be under review (as per the possible [statuses](https://docs.stripe.com/api/disputes/object#dispute_object-status)). How about we only note that a dispute was created?